### ISSUE TITLE
Add '.njk' file type support.

### ISSUE THREAD

**Issue by mrself** [robmoggach/sublime-nunjucks#11]
> Although you are free to use any file extension you wish for your Nunjucks template files, the Nunjucks community has adopted .njk.

[Nunjucks official docs](https://mozilla.github.io/nunjucks/templating.html)

**Comment by gxcodes** [comment]
Yes, I'm currently working with adonisjs which produce view templates with .njk extension. Adding this support will be very helpful.

**Comment by LantosIstvan** [comment]
I edited https://github.com/andres-risso/sublime-nunjucks cloned repo (which is works with ST3) to include `.njk` file support for myself: https://github.com/DJviolin/sublime-nunjucks

Feel free to use it if you want.

**Comment by connor11528** [comment]
@DJviolin I am getting an error downloading your package when I open sublime:

```
Error loading syntax file "Packages/Nunjucks Syntax/Nunjucks.tmLanguage": Wrong type for key begin expected string
```

**Comment by LantosIstvan** [comment]
Sorry, I'm not maintaining anymore with latest Sublime text versions. I switched to VS Code long ago.

**Comment by connor11528** [comment]
Okay cool, vs code has njk highlighting? heard a lot of good things..

https://twitter.com/Connor11528/status/924672676387479552

**Comment by LantosIstvan** [comment]
Off topic, but yeah, It's great. Much more solid then Atom and doesn't have any performance issues (this kept me to use Sublime when I started with Atom).

I use this extension for Nunjucks: https://marketplace.visualstudio.com/items?itemName=ronnidc.nunjucks

It's based on this extension.
